tests/NA12878-chr14-AKT1_1000genomes-dragen-3.7.6/run.sh

```diff
@@ -6,8 +6,9 @@
 export CAPSULE_LOG=none
 NXF_ANSI_LOG=false
 
-nextflow run ../../main.nf \
+/home/ubuntu/bin/nf24.04.2/nextflow run ../../main.nf \
```

Comment:
do not use the explicit path for your own nextflow installation. You should configure properly your environment to be able to run nextflow

```suggestion
nextflow run ../../main.nf \
```